docs/extensions/introduction.md

```diff
@@ -1,45 +1,86 @@
 ---
-description: An introduction to custom Extensions in Directus.
-readTime: 2 min read
+contributors: Rijk Van Zanten, Esther Agbaje
+description: Discover how Directus extensions enable you to expand its features and how you can customize it to suit your specific needs.
 ---
 
 # Extensions
 
-> In addition to being highly customizable, Directus has been architected to be completely modular and extensible. This
-> ensures you will never hit a hard feature ceiling within the platform.
+Extensions provide a way to build, modify or expand Directus' functionality beyond the default suite, for your specific
+needs. There are three main categories of extensions:
 
-Build, modify or expand any feature needed for your app or project. What makes Directus so flexible is that it has been
-designed from the ground up with complete extensibility. In addition to offering our software's codebase as open-source,
-we've broken down the app code into component pieces called Extensions. New Extensions can be created, modified or
-expanded to suit any need. However, the following extension types come with the platform's App and API.
+1. App Extensions
+2. API Extensions
+3. Hybrid Extensions
 
-## Extension SDK
+## App Extensions
```

Comment:
Removed the screenshots for these and moved them to `/user-guide/overview/glossary` mainly because the screenshots seemed more useful when explaining the concept of interfaces, layouts, displays, etc...not necessarily extensions.

Reply:
I think they make sense here so folks understand what type of extension they need.